test/Misc/serialized-diagnostics-batch-mode-nonprimary-errors.swift

```diff
@@ -0,0 +1,20 @@
+// Batch-mode has no good place to emit diagnostics that occur in non-primary files
+// because then cannot be locatlized to a particular primary's serialized diagnostics file.
```

Comment:
Typo: "locatlized" (and "then"). That's also not a great word because it makes me think of the internationalization localization. But the description is no longer accurate anyway.

Reply:
Fixed. I'm getting terser as the evening wears on.